ticker/include/ticker/Ticker.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef RCBE_ENGINE_TICKER_HPP
+#define RCBE_ENGINE_TICKER_HPP
+
+#include <chrono>
+#include <thread>
+#include <mutex>
+#include <functional>
+
+namespace rcbe::core {
+class Ticker {
+public:
+    using period_type = std::chrono::nanoseconds ;
+    using handler_type = std::function<void(void)>;
+    using clock_type = std::chrono::steady_clock;
+
+    Ticker() = delete;
+    explicit Ticker(std::chrono::milliseconds period);
+    ~Ticker();
+
+    void set_handler(handler_type&& handler);
```

Comment:
How about making handler a part of ctor, too?

Reply:
Made a handler a part of ctor, left this method intact, since it might be requested.